getPlaceInfo.md

```diff
@@ -0,0 +1,13 @@
+//getPlaceID.md
```

Comment:
These files are not in markdown, and they are not in the right folder. Please review the acceptance criteria